components/proxy/conf/Caddyfile

```diff
@@ -235,6 +235,13 @@ https://*.*.{$GITPOD_DOMAIN} {
 		#gitpod.workspace_port_auth {
 		#	auth_url http://server.{$KUBE_NAMESPACE}.{$KUBE_DOMAIN}:3000/auth/workspace-port/{re.host.workspacePort}
 		#}
+		# static assets configure cache headers and do not check for changes
+		@static_asset {
+			path /out/vs/*
```

Comment:
i don't think we should hard code any caching for port locations. Gitpod Code Server already does caching based on etags, we should make sure that we actually just don't intervene and don't drop/timeout requests.

Reply:
@akosyakov yes, this was just to test the behavior. Using vscode inside a workspace requires ~2000 requests every time you reload :upside_down_face: